src/TransactionTestsFiller/ttEIP1559/maxFeePerGas32BytesValueFiller.json

```diff
@@ -0,0 +1,24 @@
+{
+    "maxFeePerGas32BytesValue" :
+    {
+	    "expectException" :
+        {
+          "<=Berlin" : "TR_TypeNotSupported",
+          ">=London" : "GasLimitPriceProductOverflow"
+        },
+        "transaction" :
+        {
+            "data" : "",
+            "gasLimit" : "21000",
+            "maxFeePerGas" : "0x:bigint 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
```

Comment:
one more test case comes here. to catch GasLimitPriceProductOverflow at exact border.   so x-1 fine, x overflow, x+1 overflow. but this for another PR

Reply:
Created overflow, overflow-1 and overflow+1 tests for GasLimitPriceProductOverflow. Updating in a new branch and PR. Will close this PR now.